cpp/log/frontend_signer.cc

```diff
@@ -47,7 +48,8 @@ FrontendSigner::SubmitResult FrontendSigner::QueueEntry(
     if (sct != nullptr) {
       *sct = logged.sct();
     }
-    return DUPLICATE;
+    return Status(util::error::ALREADY_EXISTS,
+                  "Entry already exists in Database.");
```

Comment:
Those shouldn't be sentences, IMHO (no starting capital, no period). But I'm not going to be harsh on this, if you feel strongly about it... ;-)